setup.cfg

```diff
@@ -139,6 +139,7 @@ filterwarnings =
     # Ignore a warning we emit about not supporting the parallel
     # reading option for now, can be removed once the issue is fixed
     ignore:parallel reading does not currently work, so falling back to serial
+    ignore:Direct construction of DocTestModulePlus has been deprecated
```

Comment:
Either add a comment on why this was added or try to grab the latest release of `pytest-doctestplus` instead.

Reply:
OK, will do.

Reply:
p.s. I wish there was a way to tell `pip` to upgrade packages only if a new version is required by something being installed, rather than the all-or-nothing `--upgrade`.

Reply:
That's what he should do, but our requirements in pytest-astropy are not up-to-date: https://github.com/astropy/pytest-astropy/blob/master/setup.cfg#L32-L43

Reply:
Ah, I see, so I can add a comment that this can be removed once we require a more recent version of doctestplus.

Reply:
I don't see this comment added. Did I miss something?

Reply:
No, I missed something, sorry!!! Now added....